internal/log/log.go

```diff
@@ -99,10 +99,21 @@ var (
 )
 
 func init() {
-	if v := os.Getenv("DD_LOGGING_RATE"); v != "" {
-		if sec, err := strconv.ParseUint(v, 10, 64); err != nil {
-			Warn("Invalid value for DD_LOGGING_RATE: %v", err)
+	v := os.Getenv("DD_LOGGING_RATE")
+	setLoggingRate(v)
+}
+
+func setLoggingRate(v string) {
+	if v == "" {
+		return
+	}
+	if sec, err := strconv.ParseInt(v, 10, 64); err != nil {
```

Comment:
```suggestion
	if v := os.Getenv("DD_LOGGING_RATE"); v != "" {
		setLoggingRate(v)
	}
}

func setLoggingRate(v string) {
	if sec, err := strconv.ParseInt(v, 10, 64); err != nil {
```

Just a personal readability preference, where we only call `setLoggingRate` if a `DD_LOGGING_RATE` is provided. Up to you though.